### PR TITLE
Direct deposit enrollment

### PR DESCRIPTION
**Pull request by va-dittrich**
Addition of 24-0296 schema

### CODE REVIEW

File: src/schemas/24-0296/schema.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    beneficiaryAddressIsNew: {
+      type: 'boolean'
+    },
+    benefitType: {
+      type: 'string',
+      maxLength: 100
+    }
+  },
+  required: ['privacyAgreementAccepted','veteranFullName','veteranSSN','veteranVAFileNumber','veteranDOB',
```
[the remark this answers is not in the document]
Comment (va-dittrich) [reply src/schemas/24-0296/schema.js]: Will verify w/the form designer

File: src/schemas/24-0296/schema.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ['ssn','veteranSSN'],
+  ['vaFileNumber','veteranVAFileNumber'],
+  ['date','veteranDOB'],
+  ['fullName','beneficiaryFullName'],
+  ['ssn','beneficiarySSN'],
+  ['vaFileNumber','beneficiaryVAFileNumber'],
+  ['address','beneficiaryAddress'],
+  ['bankAccount','institutionAccount'],
+  ['address','institutionAddress'],
+  ['phone','institutionPhone']
```
[the remark this answers is not in the document]
Comment (va-dittrich) [reply src/schemas/24-0296/schema.js]: good catch - adding...

File: src/schemas/24-0296/schema.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (va-dittrich) [reply src/schemas/24-0296/schema.js]: using existing schemas as guide, ok to implement with something like: 
```  
anyOf: [
    {
      'required' : ['veteranSSN']
    },
    {
      'required' : ['veteranVAFileNumber']
    }
  ], 
  anyOf: [
    {
      'required' : ['beneficiarySSN']
    },
    {
      'required' : ['beneficiaryVAFileNumber']
    }
  ]
```

**Comment by ehuntdsva** [comment]
@va-dittrich  Since this is a proof of concept (and we're not doing any research), it's fine to make either SS or File Number required but not both. Also good catch on the phone number - I noticed that but forgot to add it to the spreadsheet.

**Comment by va-dittrich** [comment]
@jbalboni - all recommendations incorporated into latest commit - should be ready to go.  thanks again.

**Pull request merged**